src/common/models.ts

```diff
@@ -6,12 +6,32 @@ export interface IIssue {
   assignee: string;
 }
 
-export interface UpdateStatusPaylod {
+export interface IUsers {
+  id: number;
+  profilePicture: string;
+  name: string;
+  jobPosition: string;
+  description: string;
+  skills: Array<string>;
+}
+
+export interface UpdateStatusPayload {
   newStatus: string;
   issueId: number;
 }
 
+export interface CreateIssuePayload {
+  title: string;
+  description: string;
+  assignee: string;
+}
+
 export interface UpdateStatus {
   type: string;
-  payload: UpdateStatusPaylod;
+  payload: UpdateStatusPayload;
+}
+
+export interface CreateIssue {
```

Comment:
**UpdateStatus** and **CreateIssue** are similar interfaces. In typescript there is a way to create a generic interface and avoid duplication. Example:

```
interface Action<T> {
  type: string;
  payload: T;
}
```

I'm not sure if redux has a similar interface that you can import and use. I've seen **AnyAction**. You can check.

Reply:
done